lib/ex_pesa/Jenga/send_money_queries/account_inquiry_pesalink_banks.ex

```diff
@@ -0,0 +1,39 @@
+defmodule ExPesa.Jenga.SendMoneyQueries.AccountInquiryPesalinkBanks do
+  @moduledoc """
+  This webservice returns the recipients’ Linked Banks linked to the provided phone number on PesaLink
+  """
+
+  import ExPesa.Jenga.JengaBase
+
+  @doc """
+  Inquire about the recipients linked banks tied to a given phone number
+    Read more about Pesalink participating banks here: https://developer.jengaapi.io/reference#account-inquiry-pesalink-banks
+
+  ## Parameters
+
+  attrs: - a map containing:
+    - `mobileNumber` [string] - recipients mobile number eg 0763000000
+
+  ## Example
+      iex> ExPesa.Jenga.SendMoneyQueries.AccountInquiryPesalinkBanks.request(%{ mobileNumber: "0722000000"})
+      {:ok,
+        %{
+          "banks" => [
+            %{"bankCode" => "31", "bankName" => "Stanbic", "customerName" => "John Doe"}
+          ]
+        }
+      }
+  """
+  @spec request(map()) :: {:error, any()} | {:ok, any()}
```

Comment:
```suggestion
  @spec request(map()) :: {:error, any()} | {:ok, map()}
```

Reply:
I think if we look a the nature of responses, not all responses are always map, some are string with messages only @TraceyOnim